build-tools/biome.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Is there a place for us to specify our desired max line length? I notice that Biome seems to be line-wrapping more aggressively than prettier was. Curious is this is just a policy difference, or if we need to give it a line length to base its behavior on.

Reply:
It has a line length setting but does not seem to follow the same logic as prettier. Some of that may be due to prettier's own inconsistencies: https://biomejs.dev/formatter/differences-with-prettier/#prettier-has-inconsistencies-between-typescript-and-babel-parsing

I experimented with various settings and settled on `95` because it produced the fewest changes compared to our current formatting (I assumed we "like" the formatting we have). Reformatting is so fast it's very easy to try different settings and see their effects if you're interested in exploring yourself.